package-lock.json

```diff
@@ -24,7 +24,7 @@
         "babel-core": "6.26.3",
         "babel-eslint": "10.0.1",
         "babel-loader": "5.4.2",
-        "better-console": "0.2.4",
+        "better-console": "1.0.1",
```

Comment:
This was missed in a previous PR https://github.com/qlik-oss/qsSimpleKPI/pull/145